src/main/java/pl/edu/agh/firecell/engine/algorithm/FirePropagator.java

```diff
@@ -98,7 +98,7 @@ private int computeBurningTimeWood(Cell oldCell, double newTemperature, int curr
         return newBurningTime;
     }
 
-    private static boolean isCellBurning(Cell cell) {
+    public static boolean isCellBurning(Cell cell) {
```

Comment:
If the method is used in two sibling classes, move it to their parent. `BasicAlgorithm` would be a better place for it.